Smaller features (tsv upload, windows filenames, ...)

olehermanse opened the pull request:
Fixed #46 , replaced `*` with `X` in filenames.
Added upload of tsv (anonymous single answers), pdf (plots) and json (counted stats) - solves #47 .
Added error messages to help users when downloading reports.

olehermanse committed:
Added shortcuts for venv

Signed-off-by: Ole Herman Schumacher Elgesem <oleherman93@gmail.com>

olehermanse committed:
Updated matplotlib and pytest

Signed-off-by: Ole Herman Schumacher Elgesem <oleherman93@gmail.com>

olehermanse committed:
Remove * from paths (Fixes #46)

Signed-off-by: Ole Herman Schumacher Elgesem <oleherman93@gmail.com>

olehermanse committed:
Upload tsv, json and pdf along with html reports

Signed-off-by: Ole Herman Schumacher Elgesem <oleherman93@gmail.com>

olehermanse committed:
Useful error messages in download_reports.py

Signed-off-by: Ole Herman Schumacher Elgesem <oleherman93@gmail.com>

evestera committed:
Fix venv usage instruction

evestera commented:
Damn, github broke another line when actually performing the commit. I'll fix locally.

evestera committed:
Switch sed with perl and use hex escapes

Switch necessary because sed support for escapes is not as good in all
variants. Using hex escapes makes it easier to see what is happening and
should avoid repeats of mistakes like the previous commit.

evestera reviewed (approved):
LGTM

olehermanse commented:
Great. We should probably add some other weird unicode characters there, and a comment explanation.
